test/framework/framework.go

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Please do not create this func here. We already have two prs dealing with a common e2e test framework.  See [349](https://github.com/kubevirt/containerized-data-importer/pull/349) and [346](https://github.com/kubevirt/containerized-data-importer/pull/346)

Reply:
@jeffvance Regarding to `Framework` which has been added under `tests/` directory, do you want me to import it for functional tests (importer & controller)  in order to enable reporters?

Although I am not sure how I should use it here, because I can see that reporters are defined out of Framework structure in `tests/tests_suite_test.go` anyway .

Reply:
If I'm understanding tests setup correctly, the reporters need to be defined before the test suite is run. See [here](https://github.com/kubevirt/containerized-data-importer/blob/master/tests/tests_suite_test.go#L16)

Reply:
Yes I understand that! Although the initialization of reporters in same in all three test suites (`test/functional/controller`, `test/functional/importer`, `tests/`).

It might be good to put it in one place, as it was pointed out in comment https://github.com/kubevirt/containerized-data-importer/pull/342#discussion_r210681610 .

Do you think we can find some common place for this snippet of code, or it is ok to duplicate it ?

Reply:
Let's try not to duplicate anything - I think a good home for reporter related code would be `tests/reporters` for all dependent tests to then import in their *_suite_test.go file.

Reply:
Done